README.md

```diff
@@ -1,10 +1,18 @@
 # Awesome Neovim [![Awesome](https://awesome.re/badge-flat.svg)](https://awesome.re)
 
-<a href="https://neovim.io/"><img src="https://neovim.io/logos/neovim-mark-flat.png" align="right" width="144" /></a>
+<a href="https://neovim.io/">
+<img
+src="https://neovim.io/logos/neovim-mark-flat.png"
+align="right"
+width="144"
+/>
+</a>
 
-> Collections of awesome Neovim plugins. Mostly targeting Neovim specific features. This means Vim-compatible plugins are not listed here.
+> A collection of awesome Neovim plugins. Mostly targeting Neovim specific features.
+> This means Vim-compatible plugins are not listed here.
 
-[Neovim](https://neovim.io/) is a Vim-based text editor engineered for extensibility and usability, to encourage new applications and contributions.
+[Neovim](https://neovim.io/) is a Vim-based text editor engineered for extensibility and usability,
+to encourage new applications and contributions.
```

Comment:
I don't think this should be on a separate row

Reply:
My Neovim is wrapping after 100 lines, mb.